ibm/mas_devops/roles/suite_app_configure/vars/manage.yml

```diff
@@ -2,6 +2,21 @@
 mas_app_ws_fqn: manageworkspaces.apps.mas.ibm.com
 mas_app_ws_apiversion: apps.mas.ibm.com/v1
 mas_app_ws_kind: ManageWorkspace
-
-mas_app_cfg_delay: 480
 mas_app_cfg_retries: 50
+mas_app_cfg_delay: 480
+mas_app_ws_spec:
```

Comment:
Bad merge?  Default specs are defined in a seperate file ... see vars/defaultspecs/xxx.yaml.